backend/src/services/mod.rs

```diff
@@ -1,14 +1,33 @@
+use self::auth::{login, logout, SharedBobStore};
+use crate::{
+    connector::BobClient,
+    models::shared::Hostname,
+    prelude::*,
+    router::{ApiV1, RouterApiExt},
+};
 use axum::{
     response::{IntoResponse, Response},
     Router,
 };
-use hyper::{Body, StatusCode};
+use axum_login::RequireAuthorizationLayer;
+use hyper::{Body, Method, StatusCode};
 use thiserror::Error;
 
+pub mod auth;
+
+type RequireAuth = RequireAuthorizationLayer<Hostname, BobClient>;
+
 /// Export all secured routes
-#[allow(dead_code)]
-pub fn api_router() -> Router<(), Body> {
-    Router::new()
+///
+/// # Errors
+///
+/// This function will return an error if route pat
+pub fn api_router() -> Result<Router<SharedBobStore<Hostname>, Body>, RouteError> {
+    let router = Router::new().api_route::<_, _, ApiV1, ApiDoc>("/logout", Method::POST, logout)?;
```

Comment:
According to the documentation, `logout` is registered after `router.route_layer(RequireAuth::login())` (https://github.com/maxcountryman/axum-login#axum-example). Please, recheck that